Week2/homework/maartjes_work.js

```diff
@@ -41,7 +39,10 @@ const tuesday = [
     duration: 40
   }
 ];
-
 const tasks = monday.concat(tuesday);
-
-// Add your code here
+let twoDaysIncome = tasks
+  .map(task => task.duration / 60)
+  .filter(task => task >= 2)
+  .map(total => total * 60)
+  .reduce((a, b) => a + b, 0);
+console.log("Maartje earned " + twoDaysIncome + ".00 Euro on Monday and Tuesday!");
```

Comment:
I would look into the string method `.toLocaleString`.
If the total had contained decimal values, they would be lost with this system.